airflow/contrib/hooks/gcp_sql_hook.py

```diff
@@ -254,6 +254,54 @@ def delete_database(self, project, instance, database):
         operation_name = response["name"]
         return self._wait_for_operation_to_complete(project, operation_name)
 
+    def export_instance(self, project_id, instance_id, body):
+        """
+        Exports data from a Cloud SQL instance to a Cloud Storage bucket as a SQL dump
+        or CSV file.
+
+        :param project_id: Project ID of the project where the instance exists.
+        :type project_id: str
+        :param instance_id: Name of the Cloud SQL instance. This does not include the
+            project ID.
+        :type instance_id: str
+        :param body: The request body, as described in
+            https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/instances/export#request-body
+        :type body: dict
+        :return: True if the operation succeeded, raises an error otherwise
+        :rtype: bool
+        """
+        response = self.get_conn().instances().export(
```

Comment:
Using try..catch in case of connectivity or api issues

Reply:
Thanks, it's a valid point. However, looking at other methods in this hook and also in other GCP-related hooks we've created recently, we don't do `try`...`catch` anywhere.

Therefore I thought that maybe for the sake of coherence we could merge this "as is", and then I'd make a separate PR correcting this in all the GCP hooks?

Reply:
Let's correct this as this PR hasn't been merged yet, and then maybe raise another PR to fix others.

Reply:
Done